scss/backend/dot-admin/portlets/_content-edit.scss

```diff
@@ -33,7 +33,7 @@ $sidebar-width: 220px;
 }
 
 .content-edit__main {
-  margin: $basic-padding * 2 $sidebar-width 0 0;
+  margin: $basic-padding * 2 ($sidebar-width - 17) 0 0;
```

Comment:
What is this change for?

Reply:
![image](https://user-images.githubusercontent.com/37185433/40194862-00af564c-59c9-11e8-9495-b3001a0617b1.png)

if not then a space on the right will be added